packages/block-library/src/buttons/edit.js

```diff
@@ -1,11 +1,19 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
 /**
  * WordPress dependencies
  */
+import { __ } from '@wordpress/i18n';
 import {
 	__experimentalAlignmentHookSettingsProvider as AlignmentHookSettingsProvider,
 	InnerBlocks,
 	__experimentalBlock as Block,
+	InspectorControls,
 } from '@wordpress/block-editor';
+import { PanelBody, RadioControl } from '@wordpress/components';
```

Comment:
Personally, I prefer having imports in alphabetical order, though there's no official coding guideline one way or the other:
```suggestion
import {
	__experimentalAlignmentHookSettingsProvider as AlignmentHookSettingsProvider,
	__experimentalBlock as Block,
	InnerBlocks,
	InspectorControls,
} from '@wordpress/block-editor';
import { PanelBody, RadioControl } from '@wordpress/components';
import { __ } from '@wordpress/i18n';
```

Reply:
Implemented.